### PR TITLE
Add content for System.Runtime.Intrinsics APIs

### PR DESCRIPTION
**Pull request by mairaw**
/cc @tannergooding

### CODE REVIEW

File: xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml [hunk 1]
```diff
@@ -18,7 +18,12 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>
+             This class provides access to the Arm64 AES Crypto intrinsics
+            
+             Arm64 CPU indicate support for this feature by setting
+             ID_AA64ISAR0_EL1.AES is 1 or better
+             </summary>
```
Comment (tannergooding) [inline xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml]: nit: The indentation on many of these tags is "off"

File: xml/System.Runtime.Intrinsics.X86/Aes.xml [hunk 2]
```diff
@@ -14,7 +14,9 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>
+            This class provides access to Intel AES hardware instructions via intrinsics
```
Comment (tannergooding) [inline xml/System.Runtime.Intrinsics.X86/Aes.xml]: nit: I missed this in the seed document (and am fixing locally), but instances of `Intel * hardware instructions` should probably be changed to `x86 * hardware instructions`. 

The ISAs are available on the x86 micro-architecture from multiple CPU manufacturers (such as AMD, Intel, and Via). Explicitly listing `Intel` has given some confusion as to whether or not manufacturers other than Intel are supported (they are).

File: xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml (hunk 1 above)
Comment (mairaw) [reply xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml]: Yeah, I can fix those manually.

File: xml/System.Runtime.Intrinsics.X86/Aes.xml [hunk 3]
```diff
@@ -40,7 +42,10 @@
       <Docs>
         <param name="value">To be added.</param>
         <param name="roundKey">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>
+            __m128i _mm_aesdec_si128 (__m128i a, __m128i RoundKey)
```
Comment (mairaw) [inline xml/System.Runtime.Intrinsics.X86/Aes.xml]: are these things the name of the instructions?

File: xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml [hunk 4]
```diff
@@ -18,7 +18,12 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>
+             This class provides access to the Arm64 AES Crypto intrinsics
+            
+             Arm64 CPU indicate support for this feature by setting
+             ID_AA64ISAR0_EL1.AES is 1 or better
```
Comment (mairaw) [inline xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml]: @tannergooding what does 1 or better means here? Should it be 1 or higher instead?

File: xml/System.Runtime.Intrinsics.X86/Aes.xml (hunk 3 above)
Comment (tannergooding) [reply xml/System.Runtime.Intrinsics.X86/Aes.xml]: The first line (generally containing `_mm_*`) are the method signatures for the C/C++ version of these intrinsics.

The second line (`AESDEC xmm, xmm/m128` in this case) is the assembly instruction.

File: xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml (hunk 4 above)
Comment (tannergooding) [reply xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml]: These are generally singular bits and are architecture specific. They can generally only be `0` (not available) or `1` (available).

This is part of the stuff that will need to be explained in the "conceptual docs" with links (likely aka.ms links) to the ARM and x86 (Intel or AMD) specific architecture manuals.

Comment (tannergooding) [reply xml/System.Runtime.Intrinsics.Arm.Arm64/Aes.xml]: In this case, it looks like `ID_AA64ISAR0_EL1.AES` is 4 bits. In the latest manual it indicates:
* 0b0000 No AES instructions implemented.
* 0b0001 AESE, AESD, AESMC, and AESIMC instructions implemented.
* 0b0010 As for 0b0001, plus PMULL/PMULL2 instructions operating on 64-bit data quantities.
* All other values are reserved.

In this case, this class would be `IsSupported == true` if `AES` was `0b0001` or `0b0010`. Other values would be considered "indeterminate".

**Comment by mairaw** [comment]
@tannergooding take a look at the review site: https://review.docs.microsoft.com/en-us/dotnet/api/system.runtime.intrinsics.x86.avx?view=dotnet-plat-ext-2.1&branch=pr-en-us-2008

For sentences to be treated as paragraphs, they need to be inside \<para></para> tags.
Do you want to merge this as-is and fix this later?

**Comment by tannergooding** [comment]
> Do you want to merge this as-is and fix this later?

Yes, I think that would be fine. I am currently working on the more complete documentation now as it is.

**Comment by tannergooding** [comment]
@mairaw, what is needed from me here?

**Comment by mairaw** [comment]
@tannergooding nothing from you. I just need to take a look again to merge this.

**Pull request merged**